_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,428 @@
+
+## Assumptions
+
+This rule assumes that the auto-updating of the content is not [essential][], which is listed as valid exception to [SC 2.2.2][]. When the auto-updating of content is [essential][] this rule may produce incorrect results.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]
+- [G186: Using a control in the Web page that stops moving, blinking, or auto-updating content][g186]
+- [F16: Failure of Success Criterion 2.2.2 due to including scrolling content where movement is not essential to the activity without also including a mechanism to pause and restart the content][f16]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The text node automatically updates every 3 seconds after the page completes loading. A mechanism is available to stop the automatic updates.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="stop()" value="Stop updates" />
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+
+		function stop() {
+			clearInterval(updates)
+		}
+	</script>
+</body>
+```
+
+#### Passed Example 2
+
+The text node automatically updates every 3 seconds after the page completes loading. A mechanism is available to pause and resume the automatic updates.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="pause()" value="Pause updates" />
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		var updating = false
+		function start() {
+			var control = document.getElementById('control')
+			control.value = 'Pause updates'
+			updating = true
+			updates = setInterval(change, 3000)
+		}
+
+		function pause() {
+			if (updating) {
+				var control = document.getElementById('control')
+				control.value = 'Resume updates'
+				updating = false
+				clearInterval(updates)
+			} else {
+				start()
+			}
+		}
+	</script>
+</body>
+```
+
+#### Passed Example 3
+
+The text node (part of a progress bar) automatically updates every second after the page completes loading. Since the auto-updating can be considered essential because it takes place during a loading phase when there can be no interaction, a mechanism to stop the auto-updating is not required and the example passes.
```

Comment:
I also prefer to leave it as an assumption. I'm updating the context of the example (to remove the essential nature of the update) and moving it to a failed example (because as you said, this fails the rule), so that we have an example with a progress bar.